app/models/planet.py

```diff
@@ -0,0 +1,21 @@
+class Planet:
+    def __init__(self, id, name, description, signs_of_life=None):
```

Comment:
Since your `signs_of_life` is intended to store a boolean (I'm inferring this from the records you created), prefer `False` for the default value.

To make it more explicit that you're expecting a boolean value, a name like `has_signs_of_life` makes it more clear that this should be a yes/no value. Boolean variables/attributes often start with `has_` or `is_` (grammar-dependent) for this reason.

Reply:
Got it! change made and remembered for future!